agents-and-function-calling/bedrock-agents/features-examples/11-create-agent-using-CDK-with-knowledge-base-and-action-group/README.md

```diff
@@ -0,0 +1,135 @@
+
+# Creating Agent with Knowledge Base and an Action Group connection
+
+In this folder, we provide an example of creating an agent with Amazon Bedrock and integrating it with a 
```

Comment:
this is not the purpose of this folder.. Here we want to highlight the creation of agents using Infrastructure as code -- more specifically CDK. The example is just to show case more different capabilities. Also link to the Agents blueprint content